service-api/module/Application/src/Fixtures/DataImportHandler.php

```diff
@@ -60,4 +60,17 @@ public function importData(array $data): void
             }
         }
     }
+
+    public function insertData(string $tablename, array $item): void
+    {
+        $params = [
+            'TableName' => $tablename,
+            'Item' => $item
+        ];
+        try {
+            $this->dynamoDbClient->putItem($params);
+        } catch (AwsException $e) {
+            echo "Error: " . $e->getMessage();
```

Comment:
Should import `LoggerInterface` and use its `error` method so that it's picked up by our automated alarms